### ISSUE TITLE
Follow up message when creating a new project

### ISSUE THREAD

**Issue by garbas** [NixOS/nix#4503]
At the time of writting when you create a new project using either `nix flake new project-A` or `nix flake init` nothing is displayed to the output.

The following content should be presented to the user:
- listing all the files that were created,
- link to flake.nix schema and tutorial how to configure a flake project,
- a message (that should come from a template itself) how to use this template and what are the first steps.

/cc @edolstra

**Comment by tomberek** [comment]
For the written files?
```
diff --git a/src/nix/flake.cc b/src/nix/flake.cc
index 64fcfc000..cbd5ab1de 100644
--- a/src/nix/flake.cc
+++ b/src/nix/flake.cc
@@ -682,6 +682,7 @@ struct CmdFlakeInitCommon : virtual Args, EvalCommand
                 else
                     throw Error("file '%s' has unsupported type", from2);
                 files.push_back(to2);
+                notice("wrote: %s",to2);
             }
         };
```

Link to https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-flake.html#flake-format ? In other messages we link to the stable manual, so this would be a bit annoying.

For a "template message" there doesn't seem to be anything in https://github.com/NixOS/templates at the moment to utilize. Where would such a thing go? 
- A README.md could work at the cost of additional conflicts with existing files. 
- A README-flake.md wouldn't conflict as often
- detecting a README.md in a template and printing it when the user calls "init".

**Comment by tomberek** [comment]
- optional implementation, readFile and trace in the templates?

**Comment by tomberek** [comment]
Use welcomeText in the template schema.

**Comment by tomberek** [comment]
Fixed by #6013
